### ISSUE TITLE
cardPower - Merker für fehlende Richtungswechsel 

### ISSUE THREAD

**Issue by Armilar** [ticaki/ioBroker.nspanel-lovelace-ui#29]
![Nextion_Editor_sLpvDBLeGY](https://github.com/ticaki/ioBroker.nspanel-lovelace-ui/assets/102996011/c1868117-15a8-4fb1-a025-8f3dd06aa62a)

Elemente wie Batterie (Laden/Entladen) oder Grid (Bezug/Einspeisung) haben zwei Richtungen

**Comment by tt-tom17** [comment]
hast du für die Haus Leistung ein separaten DP?

**Comment by ticaki** [comment]
homeName

**Comment by tt-tom17** [comment]
im Adapter ist alles da, aber ich habe keinen Datenpunkt im ioBroker mit der gesamtLeistung. müsste ich errechnen mit Soloar und Netz. oder geht das intern im Adapter ??

**Comment by ticaki** [comment]
hast recht, da hab ich jetzt was unnötiges eingebaut - ich überlege mal wie ich das ausrechnen kann

**Comment by Armilar** [comment]
> hast du für die Haus Leistung ein separaten DP?

ja, führe meine komplette Energiebilanz durch unterschiedliche Räume und Gesamt

**Comment by ticaki** [comment]
In im Channel geschrieben habe ich eine option für homeValueBot eingefügt und nur dort.

```
alwaysOn: 'none',
    uniqueID: 'power1',
    useColor: false,
    pageItems: [],
    config: {
        card: 'cardPower',
        data: {
            headline: { type: 'const', constVal: 'headline' },
            homeValueTop: {
                value: { type: 'const', constVal: 'top' },
            },
            homeValueBot: {
                value: { type: 'internal', dp: '///power1/powerSum' },
                math: { type: 'const', constVal: 'return r1+r2+r3+l1+l2+l3 -999' },
            },
```

**Comment by Armilar** [comment]
Eventuell ist der ja dann mit der cardPower im Adapter abgefrühstückt

**Comment by tt-tom17** [comment]
Ich baue denn admin erstmal fertig, dann können wir über Details sprechen. Machbar ist da bestimmt einiges, Daten sind ja da.

**Comment by tt-tom17** [comment]
#184
